modules/custom/az_user_expire/config/install/user.role.az_user_expire.yml

```diff
@@ -0,0 +1,8 @@
+langcode: en
+status: true
+dependencies: {  }
+id: az_user_expire
+label: 'Auto expire user'
```

Comment:
How about Expiring User or Auto Expiring User?

Reply:
- Temporary User
- Guest User
- Short-Term Access
- Limited-Time Access

Reply:
DId "Site Access User" come up in our meeting discussion?
I'm afraid I was interrupted a few times :/
This name points out the main purpose of the user account and does not imply anything about a restricted time frame.